modules/amxBidAdapter.js

```diff
@@ -0,0 +1,279 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import { parseUrl, deepAccess, _each, formatQS, getUniqueIdentifierStr, triggerPixel } from '../src/utils.js';
+
+const BIDDER_CODE = 'amx';
```

Comment:
This does not match the https://github.com/prebid/prebid.github.io/pull/2065/files bidder code which indicates appmonet

Reply:
sorry about that! I'll update that docs PR

Reply:
fixed now

Reply:
@nickjacob please add the proper test tag and we should be good to go